GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -50,6 +50,12 @@ public class GitHubFunctions
         private readonly IAdlsClient adlsClient;
         private readonly GitHubConfigManager configManager;
         private readonly IAuditLogger ifxLogger;
+        private static volatile bool IfxLoggerInitialized = false;
+
+        // Below must match Geneva loging configuration of CloudMine.
+        // https://jarvis-west.dc.ad.msft.net/settings/onboard
+        private const string IfxTenantIdentity = "CloudMineCollectors";
+        private const string IfxRoleIdentity = "AzureDevOps";
```

Comment:
AzureDevOps?

Reply:
That was my confusion I was checking with Kivanc in our group chat earlier. It looks the config setup has to match with what was provided in Geneva (according to the comments above). But my problem is I can't find it in Geneva to verify and make sure it is okay to change the Role Identity